### ISSUE TITLE
Check for expired SSL certificates

### ISSUE THREAD

**Issue by slauger** [slauger/check_netscaler#6]
It would be nice to have a native Check (check_sslcert) which checks the expire date of all installed SSL certificates.

**Comment by slauger** [comment]
URL: http://NSIP/nitro/v1/config/sslcertkey
Field: daystoexpiration (< WARNING|CRITICAL)

**Comment by slauger** [comment]
```
[root@nagios01 check_netscaler]# ./check_netscaler.pl -H cns01 -P 'password123,.' -C check_sslcert -I sslcertkey -w 2000 -c 1000
check_netscaler CRITICAL - wildcard.tmxlab.local expires in 275 days; storefront.tmxlab.local expires in 385 days; portal.tmxlab.local expires in 252 days; wildcard.tmxlab.local expires in 225 days; mail.tmxlab.local expires in 181 days; autodiscover.tmxlab.local expires in 181 days;
```

**Comment by slauger** [comment]
Certificate check is available with v0.2.0 and enhanced with v1.0.0.
